js/lib/src/slider.js

```diff
@@ -0,0 +1,129 @@
+                this.readout.contentEditable = 'false';
+            } else {
+                this.readout.contentEditable = 'true';
+            }
+
+            let orientation = this.model.get('orientation');
+            this.$slider.slider('option', 'orientation', orientation);
+
+            // Use the right CSS classes for vertical & horizontal sliders
+            if (orientation==='vertical') {
+                this.el.classList.remove('widget-hslider');
+                this.el.classList.add('widget-vslider');
+                this.el.classList.remove('widget-inline-hbox');
+                this.el.classList.add('widget-inline-vbox');
+            } else {
+                this.el.classList.remove('widget-vslider');
+                this.el.classList.add('widget-hslider');
+                this.el.classList.remove('widget-inline-vbox');
+                this.el.classList.add('widget-inline-hbox');
+            }
+
+            let readout = this.model.get('readout');
+            if (readout) {
+                this.readout.style.display = '';
+                this.displayed.then(function() {
+                    if (that.readout_overflow()) {
+                        that.readout.classList.add('overflow');
+                    } else {
+                        that.readout.classList.remove('overflow');
+                    }
+                });
+            } else {
+                this.readout.style.display = 'none';
+            }
+        }
+
+        this.$slider.slider('option', 'range', true);
+        // values for the range case are validated python-side in
+        // _Bounded{Int,Float}RangeWidget._validate
+        let value = this.model.get('value');
+
+        this.readout.textContent = this.valueToString(value);
+
+        value = value.map(x => this.scale(x));
+        this.$slider.slider('option', 'values', value);
+    },
+
+    handleSliderChange: function(e, ui) {
+        let actual_value = ui.values.map(x => this._validate_slide_value(x));
+        this.readout.textContent = this.valueToString(actual_value);
+
+        // Only persist the value while sliding if the continuous_update
+        // trait is set to true.
+        if (this.model.get('continuous_update')) {
+            this.handleSliderChanged(e, ui);
+        }
+    },
+
+    handleSliderChanged: function(e, ui) {
+        let actual_value = ui.values.map(x => this._validate_slide_value(x));
+        this.model.set('value', actual_value, {updated_view: this});
+        this.touch();
+    },
+
+    _validate_slide_value: function(x) {
+        console.log(this)
+        return this.unscale(x);
+    },
+
+    scale: function(x) {
+        let min = this.model.get('min');
+        let max = this.model.get('max');
+
+        return (x - min) / (max - min);
+    },
+
+    unscale: function(x) {
+        let min = this.model.get('min');
+        let max = this.model.get('max');
+
+        return min + x * (max - min);
```

Comment:
@jasongrout This can fix [this issue](https://github.com/jupyter-widgets/ipywidgets/issues/259) in ipywidgets. If you like the approach I can open a PR in ipywidgets with a clean fix

Reply:
I am just scaling the jQuery slider values between 0 and 1, so that we don't trigger the issue of having slow values.